packages/ruleset/src/functions/schema-naming-convention.js

```diff
@@ -298,143 +290,6 @@ function checkSchemaNames(apidef, nodes) {
   return errors;
 }
 
-function collectResourceOrientedPaths(apidef) {
```

Comment:
All of these functions were moved to the `utils` folder for sharing between functions